examples/spark/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala

```diff
@@ -26,11 +28,15 @@ object CarbonExample {
   def main(args: Array[String]) {
     val cc = ExampleUtils.createCarbonContext("CarbonExample")
     val testData = ExampleUtils.currentPath + "/src/main/resources/data.csv"
-
+    val rootPath = new File(this.getClass.getResource("/").getPath
```

Comment:
@chenliang613 Hi liang, now the path is not get from carboncontext, they are got from carbonproperties, the code has been changed by this commit:
https://github.com/apache/incubator-carbondata/commit/f67ec0125247e98d7925866fd3062ee3ddf16e28https://github.com/apache/incubator-carbondata/commit/f67ec0125247e98d7925866fd3062ee3ddf16e28 
And you can try carbon example, its not run now.